recipes/tests.rb

```diff
@@ -341,8 +341,25 @@ module load intelmpi && mpirun --help | grep '#{node['cfncluster']['intelmpi']['
 end
 
 ###################
-# EFA - GDR (GPUDirect RDMA)
+# EFA
 ###################
+if node['cfncluster']['os'].end_with?("-custom")
+  # only check EFA is installed because when found in the base AMI we skip installation
+  execute 'check efa installed' do
+    command "modinfo efa"
```

Comment:
shall we also check that /opt/amazon/efa_installed_packages exists? without comparing the version with our installer?